include/FreeRTOS_ARP.h

```diff
@@ -57,6 +57,10 @@
         eCantSendPacket    /* 2 There is no IP address, or an ARP is still in progress, so the packet cannot be sent. */
     } eARPLookupResult_t;
 
+
+    void vARPRefreshCacheEntryAge( const MACAddress_t * pxMACAddress,
```

Comment:
Add some comment?
~~~c
/**
 * @brief Lookup an IP- and MAC-address couple in the ARP cache.
 *        If a perfect match is found, the age field will be reset.
 *        When not found, no action will be taken.
 * @param[in] pxMACAddress: The MAC-address
 * @param[in] ulIPAddress: The IP-address
 */
    void vARPRefreshCacheEntryAge( const MACAddress_t * pxMACAddress,
                                   const uint32_t ulIPAddress );
~~~